cvat/apps/consensus/consensus_reports.py

```diff
@@ -252,9 +252,12 @@ def generate_assignee_consensus_report(
         )
 
     for assignee_id, assignee_info in assignee_report_data.items():
-        assignee_report_data[assignee_id]["consensus_score"] = sum(
+        assignee_consensus_score = sum(assignee_info["consensus_score"]) / len(
             assignee_info["consensus_score"]
-        ) / len(assignee_info["consensus_score"])
+        )
```

Comment:
I think simply adding `(len(...) or 0)` would also fix the problem.

Reply:
Oh, yes. That's nice. I didn't think about this. Thanks!